### PR TITLE
copy update: /data/spark/support

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Fix incorrect copy in "Managed services" section
- Drive by:
  - Fixed indentation
  - Removed stale `canonical_url` jinja block

## QA

- View the site locally in your web browser at: https://canonical-com-1421.demos.haus/data/spark/support
- See that the text under "Go further" and "Backed by an SLA" matches the [doc](https://docs.google.com/document/d/1EcgdnMdW5jfZLLhcisUNMMtsPT-4alaPm3zirtvXEMk/edit?disco=AAABU_UlafU) 

## Issue / Card

Fixes #

### CODE REVIEW

File: templates/data/spark/support.html [hunk 1]
```diff
@@ -1,456 +1,454 @@
...
-                        </p>
-                        <hr class="p-rule--muted" />
-                        <p>
-                            <a href="https://ubuntu.com/pricing/consulting">Explore pricing&nbsp;&rsaquo;<span class="u-off-screen">explore pricing for consultancy</span></a>
-                        </p>
-                    </div>
-                </div>
+            <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+            <h3 class="p-equal-height-row__item p-heading--5">Go further</h3>
+            <p class="p-equal-height-row__item">
```
Comment (mtruj013) [inline templates/data/spark/support.html:156]: Change here

File: templates/data/spark/support.html [hunk 2]
```diff
@@ -1,456 +1,454 @@
...
+            <div class="p-equal-height-row__item p-image-container is-highlighted u-hide--small">
+              <img src="https://assets.ubuntu.com/v1/e68ed270-backed%20by%20an%20sla.png"
+                   alt=""
+                   class="p-image-container__image" />
             </div>
-        </div>
-    </section>
+            <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+            <h3 class="p-equal-height-row__item p-heading--5">Backed by an SLA</h3>
+            <p class="p-equal-height-row__item">
```
Comment (mtruj013) [inline templates/data/spark/support.html:168]: Change here

File: templates/data/spark/support.html [hunk 3]
```diff
@@ -1,456 +1,454 @@
...
-                        <hr class="p-rule--muted" />
-                        <p>
-                            <a href="https://ubuntu.com/pricing/consulting">Explore pricing&nbsp;&rsaquo;<span class="u-off-screen">explore pricing for consultancy</span></a>
-                        </p>
-                    </div>
-                </div>
+            <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+            <h3 class="p-equal-height-row__item p-heading--5">Go further</h3>
+            <p class="p-equal-height-row__item">
+              Focus on your core business by outsourcing Spark data lake house operational management. With managed services from Canonical, our engineers deploy and manage your Spark lake house solution efficiently and securely &ndash; in your cloud tenancy or physical data centre.
```
Comment (mtruj013) [inline templates/data/spark/support.html:157]: Change here

File: templates/data/spark/support.html [hunk 4]
```diff
@@ -1,456 +1,454 @@
...
+              <img src="https://assets.ubuntu.com/v1/e68ed270-backed%20by%20an%20sla.png"
+                   alt=""
+                   class="p-image-container__image" />
             </div>
-        </div>
-    </section>
+            <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+            <h3 class="p-equal-height-row__item p-heading--5">Backed by an SLA</h3>
+            <p class="p-equal-height-row__item">
+              Canonical delivers through a global, follow the sun team, with deep expertise on Spark operational service delivery. Your service is backed by industry leading compliance and SLA guarantees.
```
Comment (mtruj013) [inline templates/data/spark/support.html:169]: Change here

**Review by akbarkz — APPROVED** [review]

**Pull request merged**